examples/CRISP/client/src/design/editorial.css

```
@@ -0,0 +1,844 @@
+  --rule-strong: #0a0b0a33;
+  --accent: #0a0b0a; /* ink-as-accent for monochrome editorial */
+  --accent-tint: #a8e5bd;
+  --accent-soft: #bcedcc;
+  --warm: #0f3d2a; /* deep forest as secondary emphasis */
+  --warm-tint: #a8e5bd;
+  --warm-soft: #bcedcc;
+  --danger: #8b2a14;
+  --paper-card: #daf7e4;
+  --shadow-1: 0 1px 0 0 #0a0b0a0d;
+
+  --f-serif: 'Source Serif 4', 'Source Serif Pro', 'Iowan Old Style', Georgia, serif;
```

Stylelint (17.11.1)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)